client/src/listeners/sort-list-listener.js

```diff
@@ -0,0 +1,4 @@
+import { filterToDo } from '../handlers/sort-list.js';
+
+const filterOpt = document.querySelector('.sorting-todo');
+filterOpt.addEventListener('click', filterToDo);
```

Comment:
this works better I think:
 filterOpt.addEventListener('change', filterToDo);

Reply:
Nice suggestion, already done